tasks/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 - name: Make certain Riak is stopped
```

Comment:
I'd actually prefer it if you accepted the current PR, and then I submit another PR for changing the behaviour regarding stop/start stuff. If that's OK?
Keeps these changes separated into logical blocks by what they're changing, rather than one big-bang change.